icons/folder-seeders.svg

```diff
@@ -0,0 +1,41 @@
+<path d="m1920.5 1136.5c-11.41 2.55-23.07 4.22-35 5 11.42-2.44 23.09-4.11 35-5z" fill="#60a862"/>
+<path d="m1884.5 1141.5c-3.71 1.64-7.71 2.3-12 2 3.71-1.64 7.71-2.3 12-2z" fill="#58a35b"/>
+<path d="m1870.5 1143.5c-2.77 1.16-5.77 1.82-9 2 0.11-0.62 0.44-1.12 1-1.5 2.65-0.5 5.31-0.66 8-0.5z" fill="#66ac69"/>
+<path d="m1852.5 1146.5c-3.7 1.75-7.7 2.75-12 3 0.11-0.62 0.44-1.12 1-1.5 3.67-0.55 7.34-1.05 11-1.5z" fill="#62aa65"/>
+<path d="m1836.5 1149.5c-2.01 1.42-4.35 2.09-7 2 2.01-1.42 4.35-2.09 7-2z" fill="#67ad6a"/>
+<path d="m1827.5 1151.5c-5.15 3.04-10.82 4.37-17 4 5.48-1.64 11.15-2.98 17-4z" fill="#59a35c"/>
+<path d="m1810.5 1155.5c-2.01 1.42-4.35 2.09-7 2 2.01-1.42 4.35-2.09 7-2z" fill="#66ac69"/>
+<path d="m1799.5 1158.5c-2.01 1.42-4.35 2.09-7 2 2.01-1.42 4.35-2.09 7-2z" fill="#64aa67"/>
+<path d="m1788.5 1161.5c-5 2.44-10.33 4.11-16 5 5-2.44 10.33-4.11 16-5z" fill="#5da660"/>
+<path d="m1772.5 1166.5c-13.47 5.05-27.14 9.71-41 14 13.17-5.84 26.84-10.5 41-14z" fill="#56a159"/>
+<path d="m1106.5 1417.5c-0.18 4.21-1.51 7.87-4 11 0.96-3.89 2.3-7.56 4-11z" fill="#79b87b"/>
+<path d="m1102.5 1430.5c-2.77 9.64-6.1 18.97-10 28 2.38-9.81 5.71-19.14 10-28z" fill="#6fb271"/>
+<path d="m1085.5 1483.5c-0.4 2.99-1.4 5.65-3 8 0.1-3.14 1.1-5.8 3-8z" fill="#60a862"/>
+<path d="m1080.5 1501.5c-0.4 2.99-1.4 5.65-3 8 0.1-3.14 1.1-5.8 3-8z" fill="#5ea761"/>
+<path d="m1077.5 1512.5c-1.04 4.86-2.38 9.52-4 14 0.35-4.41 1.19-8.75 2.5-13 0.38-0.56 0.88-0.89 1.5-1z" fill="#5ba45e"/>
+<path d="m1362.5 1578.5c-2.42 9.27-5.42 18.27-9 27 2.03-9.41 5.03-18.41 9-27z" fill="#5aa45d"/>
+<path d="m1353.5 1605.5c-0.89 5.67-2.56 11-5 16 0.89-5.67 2.56-11 5-16z" fill="#5ca55f"/>
+<path d="m1348.5 1621.5c0.09 2.65-0.58 4.99-2 7-0.09-2.65 0.58-4.99 2-7z" fill="#61a863"/>
+<path d="m2199.5 1622.5c0.17 65.67 0 131.33-0.5 197-9.31 69.51-44.14 121.68-104.5 156.5-23.51 12.5-48.51 20.17-75 23-60.33 0.5-120.67 0.67-181 0.5 82.32-42.45 155.49-97.45 219.5-165 57.55-64.2 104.71-134.87 141.5-212z" fill="#388e3c" opacity=".996"/>
+<path d="m1346.5 1628.5c0.09 2.65-0.58 4.99-2 7-0.09-2.65 0.58-4.99 2-7z" fill="#63aa66"/>
+<path d="m1343.5 1640.5c0.24 2.4-0.43 4.4-2 6-0.24-2.4 0.43-4.4 2-6z" fill="#62a965"/>
+<path d="m1334.5 1676.5c0.09 2.65-0.58 4.99-2 7-0.09-2.65 0.58-4.99 2-7z" fill="#70b373"/>
+<path d="m1332.5 1685.5c0.09 2.65-0.58 4.99-2 7-0.09-2.65 0.58-4.99 2-7z" fill="#75b678"/>
+<path d="m1329.5 1698.5c0 6.29-1 12.29-3 18 0-6.29 1-12.29 3-18z" fill="#65ab68"/>
+<path d="m1326.5 1717.5c-1.25 9.53-2.92 18.87-5 28 0.62-8.73 1.79-17.4 3.5-26 0.28-0.92 0.78-1.58 1.5-2z" fill="#74b576"/>
+<path d="m1315.5 1797.5c-0.54 11.67-0.7 23.33-0.5 35 0.38-0.56 0.88-0.89 1.5-1 0.09 1.5-0.58 2.5-2 3-1.38-12.53-1.04-24.86 1-37z" fill="#77b779"/>
+<path d="m1036.5 1793.5c-0.7 38-1.04 76-1 114-0.67-15-1-30.16-1-45.5-0.17-23.02 0.5-45.85 2-68.5z" fill="#8ac48c"/>
+</svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><path d="M13.84376,7.53645l-1.28749-1.0729A2,2,0,0,0,11.27591,6H4A2,2,0,0,0,2,8V24a2,2,0,0,0,2,2H28a2,2,0,0,0,2-2V10a2,2,0,0,0-2-2H15.12412A2,2,0,0,1,13.84376,7.53645Z" style="fill:#388e3c"/><path d="M30.257,14.735A24.55713,24.55713,0,0,0,30,12a19.80379,19.80379,0,0,0-6.16046-.03407,11.06948,11.06948,0,0,0-9.43756,8.2815A22.19011,22.19011,0,0,0,14,28c3.24911.29218,6.623.27694,9.65771-1.05369C28.4316,24.99666,30.49306,19.60758,30.257,14.735Zm-4.97547,1.27753A19.11568,19.11568,0,0,0,18.403,22.80243c-.31988.56716-.65239,1.128-.94183,1.712A8.99714,8.99714,0,0,1,17.4125,21a6.82827,6.82827,0,0,1,3.75159-5.08677,8.5136,8.5136,0,0,1,5.37753-.49084A8.36484,8.36484,0,0,0,25.28149,16.01256Z" style="fill:#a4d5a6"/></svg>
</svg>
```

Reply:
Also, you're using #a4d5a6 which I believe is not included in the [material color palette](https://github.com/PKief/vscode-material-icon-theme/blob/main/CONTRIBUTING.md#use-material-design-colors). This repo has an action that lints on svg colors, so that will probably make the build fail.